clients/drcachesim/common/trace_entry.h

```diff
@@ -612,6 +612,13 @@ typedef enum {
      */
```

Comment:
>  but in the future could also be used to track changes in the vector length after prctl(PR_SVE_SET_VL, ..) system calls.

Please add a reference to the issue covering that future work.

Reply:
Done.